bridge/bridgeapp/templates/registration/login.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'home.html' %} {% block main %}
+<section>
+    {% if form.errors %}
+    <p>The input is not valid.</p>
+    {% endif %}
+    
+    {% if next %} 
+    <p>You do not have permission to access that page.</p>
+    {% endif %} 
+
+    <form method="post" action="{% url 'login' %}">
+        {% csrf_token %} 
+
+        <p>Username: {{ form.username }} </p>
+        <p>Password: {{ form.password }} </p>
+
+        <input type="submit" value="login">
+        <input type="hidden" name="next" value="{{ next }}">
+    </form>
+</section>
+{% endblock %}}
```

Comment:
I would think this page should go together with the `register.html` page, for:
1. You already don't have anything but one form that takes up the whole content of the page which is a waste of resources and poor UI/UX.
2. Both the `register.html` and `login.html` share a commonality--both take the user to the home page upon success so you're not only better utilizing the front-end but also can reuse redirect code.